replica/database.cc

```diff
@@ -2667,9 +2670,10 @@ future<> database::flush_non_system_column_families() {
 }
 
 future<> database::flush_system_column_families() {
-    auto system_cfs = get_column_families() | boost::adaptors::filtered([] (auto& uuid_and_cf) {
+    auto system_cfs = get_column_families() | boost::adaptors::filtered([this] (auto& uuid_and_cf) {
         auto cf = uuid_and_cf.second;
-        return is_system_keyspace(cf->schema()->ks_name());
+        auto& ks = cf->schema()->ks_name();
+        return is_system_keyspace(ks) || _cfg.extensions().is_extension_internal_keyspace(ks);
```

Comment:
Don't these extension-internal keyspaces depend on system ones. In other words, do we need a third stage here:
* System
* Extension
* User